### PR TITLE
Ceasing to test against non-block-scoped `stage`

### PR DESCRIPTION
**Pull request by jglick**
See https://github.com/jenkinsci/pipeline-stage-step-plugin/pull/94.

### CODE REVIEW

**Review by raul-arabaolaza — APPROVED** [review]

**Review by car-roll — APPROVED** [review]

**Comment by jglick** [comment]
```
Cannot download "https://github.com/sass/node-sass/releases/download/v4.11.0/linux-x64-64_binding.node": 
[INFO] 
[INFO] HTTP error 503 Egress is over the account limit.
```

I guess this plugin is not set up to use an NPM proxy

File: pom.xml [hunk 1]
```diff
@@ -49,7 +49,7 @@
     <argLine>-Xmx2g -Xms2g -Djava.awt.headless=true -XX:+HeapDumpOnOutOfMemoryError -XX:+TieredCompilation -XX:TieredStopAtLevel=1</argLine>
   </properties>
 
-  <scm>
+  <scm child.scm.connection.inherit.append.path="false" child.scm.developerConnection.inherit.append.path="false" child.scm.url.inherit.append.path="false">
```
Comment (olamy) [inline pom.xml:52]: why this? is it related to this PR?

File: blueocean/pom.xml [hunk 2]
```diff
@@ -15,13 +15,6 @@
     <url>https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean/doc/BlueOcean.adoc</url>
     <description>Blue Ocean is a new project that rethinks the user experience of Jenkins. Designed from the ground up for Jenkins Pipeline and compatible with Freestyle jobs, Blue Ocean reduces clutter and increases clarity for every member of your team.</description>
 
-    <scm>
```
Comment (olamy) [inline blueocean/pom.xml:18]: why this? is it related to this PR?

**Review by olamy — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (jglick) [reply pom.xml:52]: Sort of—I had problems testing this incremental build in PCT, and was hoping that fixing the deployed `scm`s might fix it.

File: blueocean/pom.xml (hunk 2 above)
Comment (jglick) [reply blueocean/pom.xml:18]: https://github.com/jenkinsci/blueocean-plugin/pull/2385#discussion_r1081778593

File: pom.xml (hunk 1 above)
Comment (jglick) [reply pom.xml:52]: Regardless of timing, this change is just matching the current recommendations: https://github.com/jenkinsci/archetypes/pull/360

**Pull request merged**